core/src/klio_core/config/_io.py

```diff
@@ -286,6 +286,12 @@ def as_dict(self):
 @attr.attrs(frozen=True)
 class KlioWriteFileConfig(KlioEventOutput, KlioFileConfig):
     file_path_prefix = attr.attrib(type=str)
+    file_name_suffix = attr.attrib(type=str, default="")
+    append_trailing_newlines = attr.attrib(type=bool, default=True)
+    num_shards = attr.attrib(type=int, default=0)
+    shard_name_template = attr.attrib(type=str, default=None)
+    compression_type = attr.attrib(type=str, default="auto")
+    header = attr.attrib(type=str, default=None)
```

Comment:
Just curious, are there any attributes here that are shared with reading a file? (and if so, perhaps they can go up in `KlioFileConfig`?)

Reply:
Looks like `compression_type` is the only shared field. I'll move that